### PR TITLE
1.19

### PR DESCRIPTION
**Pull request by basaigh**

### CODE REVIEW

File: src/main/java/com/github/steveice10/mc/protocol/packet/ingame/serverbound/inventory/ServerboundSetBeaconPacket.java [hunk 1]
```diff
@@ -7,23 +7,34 @@
...
+import javax.annotation.Nullable;
 import java.io.IOException;
 
 @Data
 @With
 @AllArgsConstructor
 public class ServerboundSetBeaconPacket implements Packet {
-    private final int primaryEffect;
-    private final int secondaryEffect;
+    private final Integer primaryEffect;
```
Comment (Camotoy) [inline src/main/java/com/github/steveice10/mc/protocol/packet/ingame/serverbound/inventory/ServerboundSetBeaconPacket.java]: Just make this an OptionalInt - I think your write method here is incorrect anyway.

Comment (basaigh) [reply src/main/java/com/github/steveice10/mc/protocol/packet/ingame/serverbound/inventory/ServerboundSetBeaconPacket.java]: Sure, sorry I didn't know those existed

Comment (basaigh) [reply src/main/java/com/github/steveice10/mc/protocol/packet/ingame/serverbound/inventory/ServerboundSetBeaconPacket.java]: Oh yes, sorry, I was going to do it a different way to begin with and forgot to change the write myself

Comment (Camotoy) [reply src/main/java/com/github/steveice10/mc/protocol/packet/ingame/serverbound/inventory/ServerboundSetBeaconPacket.java]: Mark the OptionalInts as Nonnull as well, to comply with the rest of the project.

**Pull request closed without merging**